protobuf/network.proto

```diff
@@ -57,12 +85,8 @@ message Request {
     // w/o random access to objects on server side
     repeated uint64 id = 2;
 
-    string name = 101;
-    string comment = 102;
-
-    uint64 chunk_start = 201;
-    uint64 chunk_end = 202;
-    string chunk_type = 203;
+    LocalObject object = 101;
+    ChunkDataItem chunk_item = 102;
```

Comment:
I don't particularly like how we include this in every request. Maybe we could have a RequestBody as a union based on Operation (which would be better named RequestType BTW).